packs/spell-effects/spell-effect-allegro.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "rgVQGJ0NYAt8jHh8",
+    "img": "icons/skills/movement/feet-winged-boots-blue.webp",
+    "name": "Spell Effect: Allegro",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Allegro]</p>\n<p>You perform rapidly, speeding up your ally. The ally becomes @UUID[Compendium.pf2e.conditionitems.Item.Quickened] and can use the additional action to Strike, Stride, or Step.</p>"
```

Comment:
I want to hold off adding effects that only grant conditions until we get a more sensible way to do it.